orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -179,6 +180,8 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
         transformers.emplace_back(std::make_unique<UpStreamReshapeGraphTransformer>(compatible_eps));
         transformers.emplace_back(std::make_unique<InsertGatherBeforeSceLoss>(compatible_eps,
                                                                               config.sparse_label_input_names));
+        transformers.emplace_back(std::make_unique<PaddingElimination>(compatible_eps,
```

Comment:
As synced offline, let's use an env for this new feature, before validating on more models.